### ISSUE TITLE
SC 412 -- Back arrow is read aloud in the back button

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#16]
## URL
https://demo.fileyourstatetaxes.org/en/questions/contact-preference

## Issue
The arrow graphic is being read aloud by the screen reader as it is getting inserted via CSS and impacts the accessible name of the control. 

## WCAG Success Criterion
[SC 4.1.2: Name, Role, Value (Level A)](https://w3.org/TR/WCAG21#name-role-value)

## User Impact
The accessible name of a control has to match it's function. 

### Current computed accessible name is: 
> Back arrow, Go Back 

```html
<div class="back grid" >
            <a href="/en/questions/eligible">Go back</a>
          </div>
``` 

### Should be 
> Go back

```html
<div class="back grid" aria-label="Go back">
            <a href="/en/questions/eligible">Go back</a>
          </div>
```

**Comment by aloverso** [comment]
@damiansian could you take a look at [this PR](https://github.com/codeforamerica/vita-min/pull/5869) and test the fix? I think this should resolve it. Demo link here: https://statefile.pr-5869.getyourrefund-testing.org/en

I took a different approach than what you recommended here after some exploration and chatting with folks. Adding the aria-label would work but not solve the problem at the source, instead relying on adding it anytime the component is used. My approach instead adds empty alt-text to the CSS pseudo-element so that it is indicated as decorative and prevents it from being read aloud in any context.

**Comment by damiansian** [comment]
@aloverso 

Looks great. Tested in desktop and iOS VoiceOver. 

<img width="1440" alt="code shows accessible name is derived from inner text." src="https://github.com/user-attachments/assets/763ad4d7-e240-4e68-8111-566b86eb65e8" />
